uwrt_mars_rover_hw/config/fw_config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
instead of leaving this here, i think u can have a single copy of this in the fw repo and use one of the rosinstall files to pull the specific file

Reply:
^im not sure if its possible to get it to pull a single file.

Reply:
i read up a little on rosinstall. it looks like it is possible. I just dont see the benefit.. the rosinstall cmd just copies a file from somewhere to a desired location no? isnt this the same as me just creating the file myself?

Reply:
im assuming this file is shared between the firmware and the software, right? Manual copy pasting is bad and can lead to errors. It would be better if it pulls the file at a sepcific commit to ensure that its the right version.

Its sort of like why you use git submodules instead of copy pasting a library into a repo.